<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Library_Form</class>
 <widget class="QWidget" name="Library_Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1013</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Помощник библиотекаря</string>
  </property>
  <widget class="QPushButton" name="search_name_btn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>241</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Найти по названию</string>
   </property>
  </widget>
  <widget class="QPushButton" name="search_author_btn">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>180</y>
     <width>241</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Найти по автору</string>
   </property>
  </widget>
  <widget class="QPushButton" name="append_book_btn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>111</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить книгу</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_book_btn">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>280</y>
     <width>241</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить книгу</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="search_lineedit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>491</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>481</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="radioButton_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Взятые</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Невзятые</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Все</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Фильтр книг&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="take_book_btn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>241</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Взять книгу</string>
   </property>
  </widget>
  <widget class="QPushButton" name="return_book_btn">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>370</y>
     <width>241</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Вернуть книгу</string>
   </property>
  </widget>
  <widget class="QPushButton" name="filter_btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Отфильтровать</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>10</y>
     <width>501</width>
     <height>601</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="edit_btn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>241</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Подправить информацию о книге</string>
   </property>
  </widget>
  <widget class="QPushButton" name="create_csv_btn">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>470</y>
     <width>241</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Создать csv-файл базы данных</string>
   </property>
  </widget>
  <widget class="QPushButton" name="append_books_btn">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>280</y>
     <width>121</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить книги</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
